src/MB_Toolbox.php

```diff
@@ -335,6 +335,16 @@ public function curlPOST($curlUrl, $post, $isAuth = FALSE) {
         )
       );
     }
+    elseif (strpos($curlURL, 'api.dosomething') !== FALSE) {
```

Comment:
Agreed, think of Message Broker as a third party user. The docs say `api.dosomething` so coding towards anything else such as `northstar.dosomething` seems to breaking the third party usage idea.

But thanks for the insight.